### PR TITLE
fix #77293 kontrast.dk

### LINKED ISSUE #77293: kontrast.dk
### Issue URL (Annoyance)
 [https://kontrast.dk/sektioner/podcast/artikel/respiratoren-vi-bliver-nodt-til-at-orientere-politikerne-om-hvordan-virkeligheden-er](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fkontrast.dk%2Fsektioner%2Fpodcast%2Fartikel%2Frespiratoren-vi-bliver-nodt-til-at-orientere-politikerne-om-hvordan-virkeligheden-er)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.6 nightly 22
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters, <br>Self-destructing third-party cookies (180), <br>Block Push API, <br>Block Location API
DNS filtering: | server: `https://dns-unfiltered.adguard.com/dns-query`<br>filters: `User rules`
Filters: | <b>Ad Blocking:</b><br/> EasyList, <br/>Peter Lowe's Blocklist, <br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> EasyPrivacy, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> Adblock Warning Removal List, <br/>I don't care about cookies, <br/>EasyList Cookie List, <br/>AdGuard Annoyances<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404, <br/>NoCoin Filter List
Custom filter subscriptions: | https://adguardteam.github.io/AdGuardSDNSFilter/Filters/filter.txt,<br>https://raw.githubusercontent.com/DandelionSprout/adfilt/master/LegitimateURLShortener.txt

### PR DESCRIPTION
**Pull request by wout-codes**
#77293

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -92,6 +92,9 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 lazimbana.com##.lazim-cookie-alert
+kontrast.dk###modal-background
+kontrast.dk,sued.aldi-liefert.de,enirocentrum.pl,felix1.de,goldenline.pl,panoramafirm.pl,hofer.at,dizioneri.com,m.n11.com,wuerth.de,greenweez.com,deutsches-ausschreibungsblatt.de,sued.aldi-liefert.de,capveriant.com,calkalasutasayrani.com,xn--giant-dsseldorf-5vb.de##.modal-backdrop
+kontrast.dk#$#body { overflow: auto !important; }
```
Comment (zloyden) [inline AnnoyancesFilter/sections/cookies_specific.txt:97]: For `kontrast.dk`.

```
kontrast.dk#$##modal-background { display: none !important; }
kontrast.dk#$#.modal-backdrop { display: none !important; }
kontrast.dk#$#body { overflow: visible !important; }
```

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt:97]: Sorry. I was gone for a while. Btw what's the difference? My rules work too.

Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt]: CB platforms don't have support `#$#` rules, so you can block this, but the page scrolling will be broken.
That's why we don't use `##` rules in such cases.

Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt]: Okay. Why don't the Content Blockers have JS, Sscriptlets and ext css support?

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 2]
```diff
@@ -92,6 +92,10 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 lazimbana.com##.lazim-cookie-alert
+kontrast.dk#$##modal-background { display: none !important; }
+kontrast.dk#$#.modal-backdrop { display: none !important; }
+kontrast.dk#$#body { overflow: visible !important; }
+sued.aldi-liefert.de,enirocentrum.pl,felix1.de,goldenline.pl,panoramafirm.pl,hofer.at,dizioneri.com,m.n11.com,wuerth.de,greenweez.com,deutsches-ausschreibungsblatt.de,sued.aldi-liefert.de,capveriant.com,calkalasutasayrani.com,xn--giant-dsseldorf-5vb.de##.modal-backdrop
```
Comment (zloyden) [inline AnnoyancesFilter/sections/cookies_specific.txt:98]: Also please check those domains. If they need to change the body attribute, you should remove some domains from the list.

Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt:98]: Not again.

Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt:98]: @zloyden Please evaluate this site. Another code `https://www.goldenline.pl/`

Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt]: It seems these rules are needed for `goldenline.pl`

```
goldenline.pl#$#body{ overflow: visible !important; padding-right: 0 !important; }
goldenline.pl#$?#.cookie-modal:upward(div#react-portals) { display: none !important; }
goldenline.pl#@#.cookie-modal
```

Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt]: Added.

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 3]
```diff
@@ -92,6 +92,9 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 lazimbana.com##.lazim-cookie-alert
+goldenline.pl#$#body{ overflow: visible !important; padding-right: 0 !important; }
+goldenline.pl#$?#.cookie-modal:upward(div.gl-modal) { display: none !important; }
+goldenline.pl#@#.cookie-modal
```
Comment (zloyden) [inline AnnoyancesFilter/sections/cookies_specific.txt:97]: `goldenline.pl#@#.cookie-modal` for `cookies_whitelist.txt` with task comment.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 3 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt:97]: Okay.

**Review by zloyden — APPROVED** [review]

**Pull request merged**